.github/workflows/openapi.yml

```diff
@@ -1,8 +1,6 @@
 name: openapi
 on:
-  push:
-    branches:
-      - master
+  pull_request:
```

Comment:
I'm not going to merge this change, only here so we can see the action on this PR.